examples/ogbn_papers_100m_sage.py

```diff
@@ -0,0 +1,284 @@
+# Reaches around 0.7870 ± 0.0036 test accuracy.
+
+import argparse
+import os.path as osp
+import time
+
+import torch
+import torch.nn.functional as F
+from ogb.nodeproppred import Evaluator, PygNodePropPredDataset
+from tqdm import tqdm
+
+from torch_geometric.loader import NeighborLoader
+from torch_geometric.nn import SAGEConv
+from torch_geometric.utils import to_undirected
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--device', type=str, default='cuda')
+parser.add_argument("-e", "--epochs", type=int, default=10,
+                    help='number of training epochs.')
+parser.add_argument("--runs", type=int, default=4, help='number of runs.')
+parser.add_argument("--num_layers", type=int, default=3,
+                    help='number of layers.')
+parser.add_argument("-b", "--batch_size", type=int, default=1024,
+                    help='batch size.')
+parser.add_argument("--workers", type=int, default=12,
+                    help='number of workers.')
+parser.add_argument("--neighbors", type=str, default='15,10,5',
+                    help='number of neighbors.')
+parser.add_argument("--fan_out", type=int, default=10,
+                    help='number of fanout.')
+parser.add_argument("--log_interval", type=int, default=10,
+                    help='number of log interval.')
+parser.add_argument("--hidden_channels", type=int, default=256,
+                    help='number of hidden channels.')
+parser.add_argument("--lr", type=float, default=0.003)
+parser.add_argument("--wd", type=float, default=0.00)
+parser.add_argument("--dropout", type=float, default=0.5)
+parser.add_argument(
+    "--use_undirected_graph",
```

Comment:
if this is needed for achieving the reference accuracy, I would recommend making this on by default and removing the arg parser flag.

Reply:
For performance analysis purposes, making weight-decay and dropout adjustable could benefit for our analysis.

Reply:
i meant only the use_undirected_graph part. please leave the other params you have as is